check_size.py

```diff
@@ -147,12 +167,15 @@ def check_for_updates(latest_release, releases):
     print("Found new release {}".format(latest_release))
     releases['releases'].append(latest_release)
 
-    create_report(releases['releases'])
-
     # update the releases on disc
     with open('releases_new.json', 'wb') as f:
         json.dump(releases, f, indent=4, separators=(',', ': '))
     # if everything went right, move the temp file
     shutil.move('releases_new.json', 'releases.json')
 
-    create_graph()
+
+# update report (if releases are missing from report.csv)
+create_report(releases['releases'])
```

Comment:
By simply removing a few lines from the report, they will now be recreated (good for iteration of this script)